react/src/components/Skeletons.tsx

```diff
@@ -0,0 +1,11 @@
+import { Skeleton, Stack } from "@chakra-ui/react";
+
+export function TableSkeleton() {
```

Comment:
@aerinsol I tried out a Skeleton instead of our Loading Indicator. Works quite nicely.

Reply:
@vahidbazzaz FYI